server/srv/pulp/streamer.tac

```diff
@@ -0,0 +1,66 @@
+import ConfigParser
+import logging
+import os
+import sys
+
+import mongoengine
+from twisted.application import internet, service
+from twisted.web import server
+
+from pulp.server.logs import CompliantSysLogHandler
+from pulp.server.config import config as pulp_config
+from pulp.server.db.connection import initialize as mongo_initialize
+from pulp.server.lazy.streamer import Streamer
+from pulp.plugins.loader import api as plugin_api
+
+
+DEFAULT_LOG_LEVEL = logging.INFO
+LOG_FORMAT_STRING = 'pulp_streamer: %(name)s:%(levelname)s: %(message)s'
+LOG_PATH = os.path.join('/', 'dev', 'log')
+
+
+def start_logging():
+    """
+    Configure the Pulp streamer syslog handler for the configured log level.
+    """
+    # Get and set up the root logger with our configured log level
+    try:
+        log_level = pulp_config.get('server', 'log_level')
+        log_level = getattr(logging, log_level.upper())
+    except (ConfigParser.NoOptionError, AttributeError):
+        # If the user didn't provide a log level, or if they provided an invalid one, let's use the
+        # default log level
+        log_level = DEFAULT_LOG_LEVEL
+    root_logger = logging.getLogger()
+    root_logger.setLevel(log_level)
+
+    # Set up our handler and add it to the root logger
+    if not os.path.exists(LOG_PATH):
```

Comment:
This is a slightly modified version of Pulp's logging setup. It logs to `/dev/log` which is (based on my brief research) the socket syslog listens to: http://www.gnu.org/software/libc/manual/html_node/Overview-of-Syslog.html